test/pytest/test_binary_cnn.py

```diff
@@ -37,28 +50,32 @@ def test_model2(backend, io_type):
     x = QBatchNormalization()(x)
     x = QActivation("binary_tanh", name="act4")(x)
 
-    x = QDense(10, kernel_quantizer="binary", activation="softmax", name="q_dense_7", use_bias=False)(x)
+    x = QDense(10, kernel_quantizer="binary", activation="linear", name="q_dense_7", use_bias=False)(x)
```

Comment:
Does Softmax break binary CNN?

Reply:
It doesn't break it, but the implementations differ slightly, so it's hard to get exact match. The legacy implementation got close--maybe exact. The others were off. But that didn't seem to be the focus of this PR, so I went for the simpler linear activation.